### ISSUE TITLE
Installation issues: cannot recognize TF version

### ISSUE THREAD

**Issue by fonnesbeck** [GPflow/GPflow#244]
I have the latest TensorFlow installed:

```python
In [2]: tensorflow.__version__
Out[2]: '0.11.0rc1'
```

However, trying to install GPflow from master results in the following:

```python
Collecting tensorflow>=0.11rc1 (from GPflow==0.3.3)
  Could not find a version that satisfies the requirement tensorflow>=0.11rc1 (from GPflow==0.3.3) (from versions: )
No matching distribution found for tensorflow>=0.11rc1 (from GPflow==0.3.3)
```

Running Python 3.5.2 in macOS 10.12.

**Comment by jameshensman** [comment]
Thanks for the report. 

Your setup seems very similar to mine: I also have osx/python3.5.2, but I don't get that error! Even so, it looks to me like there's a typo in the requirement, perhaps it should be 
`tensorflow>=0.11.0rc1`
in setup. py (line 47). I just fixed this in #245, though it works both with and without this fix for me. Could you check that out and report back, please?

**Comment by fonnesbeck** [comment]
Local environment issue apparently. Closing.
